### DISCUSSION TITLE
New top toolbar feels a bit crowded...

### DISCUSSION THREAD

**Discussion by llek1000** [zen-browser/desktop#1659] [Polls]
Hello, I am just pitchin in to say that, in my opinion, the old design, that is, the buttons that have now been moved to the top bar, were better in the sidebar, and I am personally really missing the new tab button at the top of the sidebar. 

Of course, this is just my personal opinion, and I am not saying that the new design is worse. I just want to get a bit more opinions from the community, and state my own opinion.

**Comment by callyral** [comment]
Now you can right click on it, customize toolbar, and move them to the sidebar. I discovered this by accident because I didn't like them in the top bar.

**Reply by callyral** [reply]
You can also hide them by dragging them back into the box where you select buttons to place.
